src/app/visit/visit.component.html

```diff
@@ -71,13 +64,21 @@
             </td>
         </ng-container>
         <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
-        <tr
-            mat-row
-            *matRowDef="let row; columns: displayedColumns"
-            (click)="clickedRow(row)"
-            [ngClass]="{ selectedRow: visitPoints.indexOf(row.id) > -1 }"
-        ></tr>
+        <tr mat-row *matRowDef="let row; columns: displayedColumns" (click)="clickedRow(row)"
+            [ngClass]="{ selectedRow: visitPoints.indexOf(row.id) > -1 }"></tr>
     </table>
     <mat-paginator [pageSizeOptions]="[10]" aria-label="Select page">
     </mat-paginator>
 </div>
+<div class="finishBar" [hidden]="!patient.name">
+    <mat-form-field class="notesOnCheck" [ngClass]="{ notesOffCheck: !checked }">
+        <mat-label>{{ 'PATIENT_VISIT.NOTES' | translate }}</mat-label>
+        <input matInput />
+    </mat-form-field>
+    <button [hidden]="!checked" class="add-btn" mat-raised-button color="primary">
```

Comment:
add a click method and print out `note` and `visitPoints` value in a console log , check **Example of the flow** in trello

Reply:
it should be printing `visitPoints`  not the full array of objects